src/coreclr/tests/issues.targets

```diff
@@ -939,6 +939,13 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- Crossgen2 on ARM64 specific -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TestBuildMode)' == 'crossgen2' and '$(RuntimeFlavor)' == 'coreclr' and ('$(TargetArchitecture)' == 'arm64' or '$(AltJitArch)' == 'arm64')">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/GitHub_27924/GitHub_27924/*">
+            <Issue>https://github.com/dotnet/runtime/issues/34316</Issue>
```

Comment:
According to the issue, it fails on x64 as well.  Why don't we add this exclusion to the previous `<ItemGroup>` then?

Reply:
Well, I only saw it on arm64 and I haven't found it in issues.targets. I'll move it to the general Crossgen2 group.

Reply:
Fixed in 15th commit. I have also sorted the Crossgen2 issues.targets items alphabetically for easier orientation.